### DISCUSSION TITLE
Unable to install on pop!_os

### DISCUSSION THREAD

**Discussion by thbeckpr** [GrandOrgue/grandorgue#2046] [Help]
Using grandorgue-wx32_3.15.2-1_arm64.deb on Ryzen 7

 grandorgue-wx32:arm64 : Depends: libasound2:arm64 (>= 1.0.16) but it is not installable
                         Depends: libc6:arm64 (>= 2.29) but it is not installable
                         Depends: libcurl4:arm64 (>= 7.28.0) but it is not installable
                         Depends: libfftw3-single3:arm64 (>= 3.3.5) but it is not installable
                         Depends: libgcc1:arm64 (>= 1:3.0) but it is not installable
                         Depends: libjack-jackd2-0:arm64 (>= 1.9.10+20150825) but it is not installable or
                                  libjack-0.125:arm64 but it is not installable
                         Depends: libstdc++6:arm64 (>= 9) but it is not installable
                         Depends: libwavpack1:arm64 (>= 4.40.0) but it is not installable
                         Depends: libwxbase3.2-1:arm64 but it is not installable
                         Depends: libwxgtk3.2-1:arm64 but it is not installable
                         Depends: zlib1g:arm64 (>= 1:1.1.4) but it is not installable

**Comment by oleg68** [comment]
You need to install amd64 on ryzen, not arm64.

**Comment by thbeckpr** [comment]
Thank you. What an embarrassing mistake!

**Comment by thbeckpr** [comment]
Resolved.
